plugin/trino-ranger/src/main/resources/conf-docker/README.md

```diff
@@ -0,0 +1,44 @@
+# Apache Ranger authorizer for Trino
```

Comment:
Can we have something like `io.trino.plugin.postgresql.TestingPostgreSqlServer` that is spins up a real Ranger docker container that is using during the CI tests, then we don't need to inject any policies via resources as we can simply call Ranger REST endpoint to create a policy. Also we don't need to inject service definition as we could simply reuse definition from Ranger service. That way we could make sure that if we use Ranger 2.5.0 client library we test it with a real Ranger 2.5.0 service and it works. Now I am afraid I don't have such certanity that this plugin works and I don't want to have a situation that this plugin would require manual verification before each release.

Also we could have something like `io.trino.plugin.postgresql.PostgreSqlQueryRunner` but for Ranger so we people could run it locally from IDE to play with Trino and Ranger.

Reply:
Tests have been updated to use the service-def packaged in in ranger-plugins-common library, instead of service-def in a local file; hope this addresses the concern of injecting service-def.

Bringing up an instance of Ranger for unit tests will be too expensive and is unnecessary to validate the plugin implementation. It would require at least 4 containers (more details at https://hub.docker.com/r/apache/ranger) and initialization of these containers can take several minutes - to create and initialize the schema. 
1. a database server (postgres/mysql)
2. a Solr instance (for audit log)
3. a zookeeper instance (required by Solr)
4. a Ranger instance

> Now I am afraid I don't have such certainty that this plugin works and I don't want to have a situation that this plugin would require manual verification before each release.

It will help if you can add more details of the situations you are concerned about.

> Also we could have something like io.trino.plugin.postgresql.PostgreSqlQueryRunner but for Ranger so we people could run it locally from IDE to play with Trino and Ranger.

@kokosing - thank you for the pointer to `TestingPostgreSqlServer`. I see that this brings up Postgres server in a container using `PosgreSQLContainer`. A similar approach can be used to bring up Ranger as well, but this will require significant development effort to automate - with no additional benefit to existing unit tests. Can this support be tracked in a separate issue, subsequent to this PR? In the meantime, anyone interested in playing with Ranger instance can bring it up in docker container using instructions at https://hub.docker.com/r/apache/ranger.

Reply:
I see. As I understand we could say that these test is not about testing Ranger client library to see if the audit information is actually generated etc. We only want to see if the usage of library works, and we assume that library is tested elsewhere. Fair enough.

Reply:
yes, that is correct.